main.go

```diff
@@ -124,6 +125,22 @@ func (m *metricProcessor) authenticate() (err error) {
 	return nil
 }
 
+func updateAppSpaceData(app *cfclient.App) error {
+	if (cfclient.SpaceResource{}) == app.SpaceData {
```

Comment:
I found this a bit confusing to read. Traditionally the thing you're testing is on the left of the comparison rather than the right. Could you change it to:
```
if app.SpaceData == cfclient.SpaceResource{} {
```